kf_lib_data_ingest/etl/extract/operations.py

```diff
@@ -81,6 +86,8 @@ def value_map(m, in_col, out_col):
 
     def value_map_func(df):
         new_df = DataFrame()
+        if optional and (in_col not in df):
```

Comment:
Shouldn't there be an error if the operation is NOT optional (required) and the `in_col` is not found?

Reply:
there will be